### PR TITLE
Doc fixes

### PR DESCRIPTION
**Pull request by fbertsch**
Just a few doc fixes I noticed while reading through them.

- [X] Add documentation.
- [ ] Add a changelog entry.
- [ ] Add your name in the contributors file.

### CODE REVIEW

**Comment by glasserc** [comment]
Thanks! I'm going to wait until we sort out the failing tests (see #2020) before merging, should be sometime tomorrow.

File: docs/api/1.x/filtering.rst [hunk 1]
```diff
@@ -101,8 +101,8 @@ The filters ``lt`` and ``gt`` are available to compare against values.
   than 100.
 
 This bound is exclusive (i.e., in the previous example, it would not
-retrieve a record whose ``orders`` field was equal to 100. To check
-"less than or equal", use ``min``. To check "greater than or equal",
+retrieve a record whose ``orders`` field was equal to 100). To check
+"greater than or equal", use ``min``. To check "less than or equal",
```
Comment (leplatrem) [inline docs/api/1.x/filtering.rst]: :woman_facepalming:

**Comment by Natim** [comment]
@fbertsch can I let you add your name in the contributors file and a litle line in the Changelog?

**Comment by fbertsch** [comment]
@Natim done and done

**Pull request merged**